.github/workflows/scheduled-prod-deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: Scheduled Prod Deploy
+
+on:
+  schedule:
+    # Weekly Mondays & Thursdays @ 15:00 UTC (8am PST)
+    - cron: '0 15 * * 1,4'
+
+env:
+  DOCKER_BUILDKIT: 1
+  COMPOSE_DOCKER_CLI_BUILD: 1
+  DOCKER_REPO: ${{ secrets.ECR_REPO }}/
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  upgrade:
+    runs-on: ubuntu-20.04
+    environment:
+      name: prod
+    steps:
+      - uses: act10ns/slack@v1
+        with:
+          status: starting
+        if: always()
+      - name: Configure AWS Prod Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_PROD_ROLE_TO_ASSUME }}
+          role-duration-seconds: 900
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - run: git checkout HEAD^
```

Comment:
I think the actions/checkout step above clones the repo at the head of the default branch - do we need this step?

Reply:
nope, I saw that too. got rid of it